sw-precache-config.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
     '/data/**/*'
   ],
   navigateFallback: '/index.html',
+  directoryIndex: 'index.html',
```

Comment:
it was but seems like it doesn't set by default now. That causes the issue with fetching files from the cache